server/pkg/key/key.go

```diff
@@ -0,0 +1,34 @@
+package key
+
+import (
+	"regexp"
+
+	"github.com/goombaio/namegenerator"
+	"github.com/reearth/reearth-cms/server/pkg/util"
+)
+
+var keyRegexp = regexp.MustCompile("^[a-zA-Z0-9_-]{5,32}$")
+
+type Key struct {
+	key string
+}
+
+func NewKey(key string) Key {
```

Comment:
```suggestion
func New(key string) Key {
```

For users of this package, `key.New` is sufficient to undertstand